packages/ag-charts-community/project.json

```diff
@@ -125,6 +125,14 @@
       "options": {
         "commands": ["node tools/package/sanity-check-package.js dist/packages/contents/ag-charts-community/package"]
       }
+    },
+    "validate-size": {
```

Comment:
We should either tie this into the `package:verify` or `test` tasks as a dependency so that it is run routinely.

Reply:
I think it should be tied to the tests, question is do we already want to do it or should we wait until there's actually something to test?
There's also a question whether we should already add this to the `core` and `enterprise` packages.